pymongoexplain/__main__.py

```diff
@@ -48,6 +49,18 @@ def new_func(self: Collection, *args, **kwargs):
     setattr(Collection, old_func_name, make_func(old_func, old_func_name))
 
 if __name__ == '__main__':
-    for file in sys.argv[1:]:
+    parser = argparse.ArgumentParser()
+    parser.add_argument(
```

Comment:
Can we add helptext here that provides:
* a 1-2 line summary of what this script is intended to do
* a link to more detailed documentation - in this case this can just be the GH repo as the 'docs' are the readme.

See https://docs.python.org/3/library/argparse.html#add-help for how to add helptext.

Reply:
Done.